server/src/lib/rpc/client.js

```diff
@@ -40,4 +41,14 @@ rpc.nav.getNewAddress = async () => {
   }
 }
 
+rpc.nav.listUnspentTx = async () => {
```

Comment:
Why do we need this?
Can't we just use `rpc.listUnspent()` directly? Why does it need a wrapper?